src/common/config_parser.cpp

```diff
@@ -701,11 +705,42 @@ void ConfigParser::expandAliases(cli::CLIWrapper& cli) {
     config["dec-cell-high-depth"] = 2;
     config["dec-depth"] = 4;
     config["skip"] = true;
-  }
-
-  if(config) {
-    cli.updateConfig(config, "Unknown option(s) in aliases");
-  }
+  });
+
+  cli.alias("task", "transformer", [](YAML::Node& config) {
```

Comment:
I guess these should also have some comments, I guess this is something I can do later.